enhancements/etcd/disaster-recovery-with-ceo.md

```diff
@@ -0,0 +1,97 @@
+---
+title: disaster-recovery-with-ceo
+authors:
+  - "@skolicha"
+reviewers:
+  - "@deads2k"
+  - "@hexfusion"
+  - "@alpatel07"
+approvers:
+  - TBD
+creation-date: 2020-02-19
+last-updated: 2020-02-19
+status: provisional
+see-also:
+  - "https://github.com/kubernetes/enhancements/blob/master/etcd/cluster-etcd-operator.md"  
+---
+
+# disaster-recovery-using-cluster-etcd-operator
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Current disaster recovery scripts were created before the introduction of `cluster-etcd-operator` (CEO). Cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd during cluster bootstrap and regular operation and provisioning etcd dependencies such as TLS certificates. This enhancement allows modifications to the DR scripts such that disaster recovery makes use of the presence of CEO.
+
+
+## Motivation
+
+The immediate need for enhancement of disaster recovery scripts is to allow basic backup and restore operations to work, which are currently failing the automated tests (e2edisruptive tests). 
+
+In addition, the disaster recovery should allow the recovery of a single failed node, scaling up and scaling down (node drain).
+
+## Goals
+
+*  Backing up etcd data, including etcd static-pod-resources
+*  Restoring to a previous state
+*  Recover a single failed node
+   *  For a new node
+   *  For an existing node
+*  Node Drain
+
+## Non-Goals
+
+*  Recover from expired certificates (not considered for 4.4)
+
+## Proposal
+
+## User Stories [optional]
+
+## Implementation Plan
+
+1. Make changes to the scripts as needed
+2. Modify CEO to provide the information needed for the DR scripts to work.
+
+## Implementation Details/Notes/Constraints
+
+*  Fix the version mismatch bug
+*  Change the name of the spec from `etcd-member.yaml` to `etcd-pod.yaml`
+*  Change the backup scripts to include the static pod resources of etcd
```

Comment:
Can you write out the logical steps required for our cases?  Most people reading aren't familiar with the vagaries of the current script and it such text age well.